src/kedarKDec19/Assignment2/Question1.java

```diff
@@ -0,0 +1,39 @@
+package kedarKDec19.Assignment2;
+
+public class Question1 {
+	
+	/*
+	 * Print missing number from 1 to 10 
+	 * 
+	 * 
+	 * 
+	 */
+
+	static void missing(int[]a) {
+		
+		for (int i=1;i<=10;i++) {
+			
+			String found ="n";
+		 
+			for (int j=0;j<a.length;i++) {
+				
+				if (i==a[j]) {
```

Comment:
very poor coding style.. no one can understand this "Y" and "N" concept?
what is use case of this?

Reply:
Logic is correct but == will fails always.
check with flag value